### PR TITLE
fix(multiple): style changes for better accessibility

### PR DESCRIPTION
**Pull request by mhaertwig**
* accordion: corret color of header-indicator icon
* chips: use cursor pointer for draggable chips
* notification: change color of warn notification in lean mode
* status: change color and icon of warn state and icon of invalid state
* time-input: center text for disabled input in standard mode

### CODE REVIEW

File: src/angular/styles/_typography.scss [hunk 1]
```diff
@@ -1764,6 +1764,10 @@ textarea {
     line-height: sbb.pxToRem(20);
   }
 
+  .sbb-checkbox & {
```
Comment (kyubisation) [inline src/angular/styles/_typography.scss:1767]: @mcilurzo Are the text colors for checkbox and radio button intentionally different?

**Review by kyubisation — CHANGES_REQUESTED** [review]

**Comment by jeripeierSBB** [comment]
Additionally, I had im my mind, that every usage of color grey should be replaced by granite ?

File: src/angular/styles/_typography.scss (hunk 1 above)
Comment (mhaertwig) [reply src/angular/styles/_typography.scss]: I've started a discussion in our Teams channel and temporary removed this code, Later I'll make a new PR for the checkbox and radio button changes.

**Review by kyubisation — APPROVED** [review]
LGTM 👍 
Thank you very much!

File: src/angular/accordion/expansion-panel.scss [hunk 2]
```diff
@@ -83,7 +83,7 @@
   :where(html:not(.sbb-lean)) & {
     border: var(--sbb-border-width-thin) solid var(--sbb-color-storm);
     border-radius: 50%;
-    color: var(--sbb-color-granite);
+    color: var(--sbb-color-black);
```
Comment (jeripeierSBB) [inline src/angular/accordion/expansion-panel.scss:86]: I see that you re-added it, but I think then we should apply it for lean also? Or should it be placed on a parent container?

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Pull request merged**